streamwebs_frontend/streamwebs/models.py

```diff
@@ -29,19 +30,22 @@ class Site(models.Model):
     location as a pair of latitudinal/longitudinal coordinates and an optional
     text description of entry.
     """
-    site_name = models.CharField(max_length=250, blank=True)
+    site_name = models.CharField(
+        max_length=250, blank=True, verbose_name=_('site name')
+    )
     site_type = models.CharField(max_length=250, blank=True)
     description = models.TextField(blank=True)
     site_slug = models.SlugField(blank=True)
 
     # Geo Django fields to store a point
     location = models.PointField(null=True, blank=True)
-    objects = models.GeoManager()
+    # objects = models.GeoManager()
 
     created = models.DateTimeField(default=timezone.now)
     modified = models.DateTimeField(default=timezone.now)
 
     test_objects = SiteManager()
+    objects = test_objects
```

Comment:
Set ``object = models.Manager()``. SiteManager() only creates objects for tests, so we'd be losing a lot of functionality using it as our only manager for the site model.